addons/web/models/models.py

```diff
@@ -279,6 +340,8 @@ def search_panel_select_multi_range(self, field_name, **kwargs):
         comodel_domain = kwargs.get('comodel_domain', [])
         disable_counters = kwargs.get('disable_counters', False)
 
+        Comodel = self.env.get(field.comodel_name)
+
```

Comment:
This looks like noise.  Is this a real change?

Reply:
no but this was not at the right place before I think